### PR TITLE
Client side password rule validation doc

### PR DESCRIPTION
**Pull request by mooreds**
Pulling over the gist Daniel wrote into the example scripts repo and documenting it.

When this is published, we should update https://fusionauth.io/community/forum/topic/2383/checking-password-rules-before-submission

### CODE REVIEW

**Review by mark-robustelli — APPROVED** [review]
Looks good to me.

File: site/docs/v1/tech/themes/client-side-password-rule-validation.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+FusionAuth checks password rules in server-side validation, but you can also check them in your client. This page shows one method of doing so in JavaScript.
+
+{% comment %}
+This page is primarily pulled from a GH subdirectory, ensuring that the code can be ran and updated easily. The README is also in that repo to make sure it stays in sync with the code.
+
+When you change the remote files, there may be a 5 minute or so lag after your push before the changes are available at the `raw` URL, so don't be surprised if your changes don't immediately appear.
+{% endcomment %}
+
+{% remote_include https://raw.githubusercontent.com/FusionAuth/fusionauth-example-scripts/master/client-side-password-rules/README.md %}
```
Comment (spwitt) [inline site/docs/v1/tech/themes/client-side-password-rule-validation.md:16]: This file includes link to our documentation pages. Is that going to create a headache when we migrate docs? Then we will have external resources that are linking to our doc that may not be picked up during the doc migration and will produce dead links.

**Review by spwitt — APPROVED** [review]
Maybe we'll have some sort of link translation after doc migration that will resolve old bookmarks and things, and this won't be an issue.

File: site/docs/v1/tech/themes/client-side-password-rule-validation.md (hunk 1 above)
Comment (mooreds) [reply site/docs/v1/tech/themes/client-side-password-rule-validation.md:16]: Rewrites are a critical part of our doc migration :) . No link left behind!

**Comment by jobannon** [comment]
@mooreds I am wondering if it makes sense to drop this link https://fusionauth.io/articles/security/breached-password-detection#what-does-nist-have-to-say-about-breached-password-detection in this page or adjacent pages.
Just a thought.

**Comment by mooreds** [comment]
@jobannon I think we do: https://fusionauth.io/docs/v1/tech/themes/client-side-password-rule-validation shows that link.

Great minds think alike!

**Pull request merged**